docs/user-guide/mvd-editor.md

```diff
@@ -2,40 +2,71 @@
+## Specifying a highlighting language
 
-To specify a language server, complete these steps:
+1. Click **Language** on the editor toolbar. A dropdown menu will be displayed.
+2. From the dropdown, select the desired language. Plain Text will be chosen by default.  
 
-1. From the **Language Server** menu, select **URL***.
-2. From the **Language Server Setting**, **Put your config here** area, paste your configuration.
-3. Ensure that the **Enable Language Server** check box is selected.
-4. Click **Save**.
+## Open a dataset
 
-## Specifying a language
+To open a dataset, follow these steps:
 
-From the **Language** menu, select the language you want to use.
+1. From the **File** menu, select Open Datasets. You can also use (ALT+K).
+2. In the Dataset field, specify the name of the dataset you want to open. 
+3. Click **Open** 
+
+## Deleting a dataset
+
+To delete a dataset, follow these steps:
+
+1. In the file tree, right-click on a dataset you want to delete.
+2. From the right-click menu, click **Delete**. A warning dialogue will appear. 
+3. Click **Delete** 
 
 ## Opening a directory
 
 To open a directory on the system, complete these steps:
 
-1. From the **File** menu, select **Open Directory**.(Alternatively, you can click **Open Directory** in the File Explorer.)
-
-2. From the **Open Directory**, **Input Your Directory** field, type the name of the directory you want to open. For example: `/u/zs1234`
+1. From the **File** menu, select **Open Directory**. You can also use (ALT+O).
+2. In the Directory field, specify the name of the directory you want to open. For example: `/u/zs1234`
 3. Click **Open**.
 
 The File Explorer on the left side of the window lists the folders and files in the specified directory. Clicking on a folder expands the tree. Clicking on a file opens a tab that displays the file contents.
 
+## Creating a new directory
+
+1. Right-click on a location in the directory tree where you want to create a new directory.
+2. From the right-click menu, click **Create a directory...**.
+3. Specify a directory name in the Directory Name field. 
+4. The Path will be set to the location that you intially right-clicked to open the dialogue. You can specify a different location in the Path field. 
+5. Click **Create**    
+
 ## Creating a new file
 
 To create a new file, complete these steps:
 
-1. From the **File** menu, select **New File**. The **New File** tab opens.
-2. From the **New File**, **File Name** field, type the name of the file.
-3. Click **Create**.
+1. From the **File** menu, select **New File**. You can also use (ALT+N).
+2. From the **File** menu, select **Save** to save the newly created file. You can also use (Ctrl+S)
+3. In the File Name field, specify the file name for the newly created file.
+4. Choose an encoding option from the Encoding dropdown menu. The directory will be prefilled if you are creating the new file in an existing folder.
+5. Click **Save**
+   
 
 ## Saving a file
```

Comment:
This category is summarized in the above "Creating a new file" category, so you can completely remove it

Reply:
You can also add a step 6 if you'd like

6. To close a file, click on the X icon in its tab, double-click on the tab, or use (Alt+W).